content/en/releases/core/_index.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Core Framework Releases"
+linkTitle: "Core Framework"
+weight: 2
+description: >
+  Versions currently supported, dependencies, and release notes for the CHT Core Framework
+no_list: true
+aliases:
+  -    /core/overview/supported-software
```

Comment:
We'll need to add aliases to ever page to be redirected from the old URL to the new URL.  Here's an example on this page.  It's original URL was` https://docs.communityhealthtoolkit.org/core/releases/` but the new URL is `https://docs.communityhealthtoolkit.org/releases/core/`, so we'll add a new alias like this:

```suggestion
aliases:
  -    /core/releases
  -    /core/overview/supported-software
```